src/index.js

```diff
@@ -89,13 +92,24 @@ class Select extends Component {
 
   componentDidMount() {
     document.addEventListener('mousedown', this._handleDetectClickOutside)
+    window.addEventListener('scroll', this.handleChangeOptionPosition)
   }
 
   componentWillUnmount() {
     document.removeEventListener('mousedown', this._handleDetectClickOutside)
   }
   /* End of Detect click Outside */
 
+  handleChangeOptionPosition = () => {
+    const windowHeight = window.innerHeight;                                                    // current window height
+    const componentTop = this.component.getBoundingClientRect().top;                            // select component offset top
+    const optionHeight = this.menuOption && this.menuOption.offsetHeight + 50 || 400;           // option component height
+    const optionPosition = componentTop + optionHeight >= windowHeight ? 'dropup' : 'dropdown'; // option position
+    this.setState({
+      optionPosition: optionPosition,
+    })
+  }
+  
```

Comment:
เสนอให้สามารถ ตั้ง propTypes กำหนด position ที่เปิด option ได้
เผื่อคนที่ต้องการ fix ตำแหน่งการเปิดเลย
- dropup
- dropdown
- auto (เรียก this.handleChangeOptionPosition) <-- default